libs/features/google-login.js

```diff
@@ -3,10 +3,12 @@ const GOOGLE_ID = '530526366930-l874a90ipfkn26naa71r010u8epp39jt.apps.googleuser
 const PLACEHOLDER = 'feds-googleLogin';
 const WRAPPER = 'feds-profile';
 
-const onToken = async (getMetadata, data) => {
+const onToken = async (getMetadata, data, getConfig) => {
   let destination;
+  const config = getConfig();
   try {
-    destination = new URL(getMetadata('google-login-redirect'))?.href;
+    destination = new URL(typeof config.googleLoginURLCallback === 'function' ? await config.googleLoginURLCallback()
```

Comment:
@vhargrave In testing this further with Haris, I noticed you wouldn't ever get to getMetaData() as a fallback as the googleLoginURLCallback doesn't handle that scenario. I think we need it to look for the metadata in the callback in the event of nothing to return.